### ISSUE TITLE
[ED-Low] 'Optional' throughout

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-curricula#146]
Throughout the resources in the Teaching Ideas, Ideas to Assess Knowledge and Ideas to Assess Knowledge sections there is an opening sentence of the form:

> Optional ideas to teach the learning outcomes

or

> Optional ideas to support assessment

I was wondering what the point of these were. The only thing they seemed to add was that these were 'optional' - which is more to say that instructors could pick and chose how they might use the various different items. However, I am not sure that it does it that well.

Also, it struck me as peculiar that everything is optional. If there are learning outcomes that students should attain then surely there are things that should be delivered for them to attain those outcomes.

**Comment by daniel-montalvo** [comment]
Hi @iadawn 

A while ago we had a paragraph just below the "Topics to Teach" section saying:

> The following topics help achieve the learning outcomes. You might choose to adapt the sequence of these topics, the activities you carry out, and the homework you provide according to the particular audience and context.
Which aim was to communicate what you said: instructors could pick what they need or find useful.

That was taken out because the majority thought it was too worthy and did not add much.

We could discuss this again for future iterations, anyway.

Some proposals:

<blockquote>

## Teaching Ideas

Choose the ones needed to deliver the learning outcomes.

Include the most appropriate to deliver the learning outcomes.

## Ideas to Assess Knowledge

Choose the ones needed to assess the learning outcomes.

Include the most appropriate to assess the learning outcomes.

</blockquote>

**Comment by iadawn** [comment]
Agreed that the original is too long for not enough info. The alternative proposals seem to work better. They make it clearer that this is a pick and mix.

**Comment by daniel-montalvo** [comment]
Good to know that they sound better for you @iadawn 

We discussed the title of these sections, especially "Ideas to Assess Knowledge" in [EOWG 27 September call](https://www.w3.org/2019/09/27-eo-minutes.html#item03) so changing these sentences might be not a good idea before bringing it back to the group for discussion.

Changed in 7f44125 and in 3600d2a to come back to this later.

**Comment by iadawn** [comment]
Wasn't so much the title as the opening line :)

**Comment by nitedog** [comment]
I feel that wording like "chose" or "include" seem too restrictive. Depending on the particular course and instructor, maybe none of the suggested ideas apply. Maybe not even variations of these ideas, and instructors need to come up with their own ideas to teach a particular learning outcome. So maybe @iadawn's understanding of "pick and mix" is different. It is more "get inspired and mix".

**Comment by iadawn** [comment]
I might be missing your point @nitedog but I feel that if there are learning outcomes that none of the teaching ideas relate to then there is a gap. This might be a more involved debate :)

**Comment by daniel-montalvo** [comment]
I think we do have teaching ideas that relate to at least one learning outcomes, so hopefully we might not have that gap. Do you agree with this @iadawn 

So to balance and not be so restrictive, we could have:

<blockquote>

## Teaching Ideas

Base your 
</blockquote>

**Comment by daniel-montalvo** [comment]
Please have a look at https://github.com/w3c/wai-curricula/issues/146#issuecomment-560328325 for an edited version of the previous comment, which was uncompleted.
